kuma/static/js/analytics.js

```diff
@@ -129,30 +140,40 @@
             this article was a lot of help: http://blog.gospodarets.com/track_javascript_angularjs_and_jquery_errors_with_google_analytics/
         */
         trackClientErrors: function() {
-
             // javascript & jQuery errors
             $(win).on('error', function(e) {
                 // probably javascript
-                if(e.originalEvent) {
+                if (e.originalEvent) {
                     var originalEvent = e.originalEvent;
-                    var lineAndColumnInfo = originalEvent.colno ? ' line:' + originalEvent.lineno +', column:'+ originalEvent.colno : ' line:' + originalEvent.lineno;
-                    analytics.trackError('JavaScript Error', originalEvent.message , originalEvent.filename + ':' + lineAndColumnInfo);
-                }
-                // no originalEvent means probably jQuery
-                else {
+                    var lineAndColumnInfo = originalEvent.colno
+                        ? ' line:' +
+                          originalEvent.lineno +
+                          ', column:' +
+                          originalEvent.colno
+                        : ' line:' + originalEvent.lineno;
```

Comment:
This is unreadable spread over multiple lines like this. I think is just too long to go into a ternary function.